hieradata/dev/wso2/wso2is/pattern-2/default.yaml

```diff
@@ -173,42 +185,91 @@ wso2::bps_datasources:
     validation_query: "%{hiera('wso2::datasources::mysql::validation_query')}"
     validation_interval: "%{hiera('wso2::datasources::common::validation_interval')}"
 
-wso2::metrics_datasources:
-  metrics_ds:
-    name: WSO2_METRICS_DB
-    description: The default datasource used for WSO2 Carbon Metrics
+wso2::am_datasources:
+  wso2_am_db:
+    name: WSO2AM_DB
+    description: The datasource used for API Manager database
     driver_class_name: "%{hiera('wso2::datasources::mysql::driver_class_name')}"
     url: jdbc:mysql://192.168.100.1:3306/WSO2_CONFIG_DB?autoReconnect=true
     username: "%{hiera('wso2::datasources::mysql::username')}"
     password: "%{hiera('wso2::datasources::mysql::password')}"
-    jndi_config: bpsds
+    jndi_config: jdbc/WSO2AM_DB
     max_active: "%{hiera('wso2::datasources::common::max_active')}"
-    max_idle: "%{hiera('wso2::datasources::common::max_idle')}"
     max_wait: "%{hiera('wso2::datasources::common::max_wait')}"
     test_on_borrow: "%{hiera('wso2::datasources::common::test_on_borrow')}"
-    use_datasource_factory: "%{hiera('wso2::datasources::common::use_datasource_factory')}"
     default_auto_commit: "%{hiera('wso2::datasources::common::default_auto_commit')}"
     validation_query: "%{hiera('wso2::datasources::mysql::validation_query')}"
     validation_interval: "%{hiera('wso2::datasources::common::validation_interval')}"
 
-wso2::am_datasources:
-  wso2_am_db:
-    name: WSO2AM_DB
-    description: The datasource used for API Manager database
+wso2::metrics_datasources:
+  metrics_ds:
+    name: WSO2_METRICS_DB
+    description: The default datasource used for WSO2 Carbon Metrics
     driver_class_name: "%{hiera('wso2::datasources::mysql::driver_class_name')}"
     url: jdbc:mysql://192.168.100.1:3306/WSO2_CONFIG_DB?autoReconnect=true
     username: "%{hiera('wso2::datasources::mysql::username')}"
     password: "%{hiera('wso2::datasources::mysql::password')}"
-    jndi_config: jdbc/WSO2AM_DB
+    jndi_config: bpsds
```

Comment:
AFAIU, this is the metrics DB entry, in which case,

1. Do we need to offload it to an external DBMS?
2. Shouldn't the JNDI Name change to reflect it?